### DISCUSSION TITLE
UI component framework policy

### DISCUSSION THREAD

**Discussion by losolio** [losol/eventuras#735] [General]
We were using Chakra UI for prototyping, but switched to Mantine.dev. We should reduce dependencies, and bundle size, and could try to make more ui components ourself. There are some prioritized UI components

Features: 
* Should integrate well with Next.JS
* Bundle size matters
* Accesibility
* Good look

Components of particular interest
* Date picker 
* Drawer
* Modal
* Forms
* Notifications/toaster

**Comment by losolio** [comment]
- [Chakra UI](https://chakra-ui.com/) - misses datepicker
- [Mantine.dev](https://mantine.dev) - looks good
- [Radix](https://www.radix-ui.com/case-studies) 
- [Shadcn/ui](https://ui.shadcn.com/) - built on Radix and Tailwind

**Comment by meinte** [comment]
I like the heading we are currently taking - wrap browser native form elements for MVP and then afterwards we can see when we hot those limitations
